src/Web/Avalonia.Web/WindowingPlatform.cs

```diff
@@ -101,5 +94,25 @@ private static IRuntimePlatform GetRuntimePlatform()
         {
             return AvaloniaLocator.Current.GetRequiredService<IRuntimePlatform>();
         }
+
+        Size IPlatformSettings.GetTapSize(PointerType type)
```

Comment:
Looks like it's just a copy paste of default one.

Reply:
Yep, obviously missed removing that when I added `DefaultPlatformSettings`.